src/app/space/settings/collaborators/add-collaborators-dialog/add-collaborators-dialog.component.less

```diff
@@ -1,4 +1,5 @@
 @import (reference) '../../../../../assets/stylesheets/shared/osio.less';
+@import '../../../../../../node_modules/@ng-select/ng-select/themes/default.theme.css';
```

Comment:
Does less duplicate the rules if another less file imports the same css?

Reply:
Webpack should remove duplicate rules if another file imports the same css i think.

Reply:
I feel like that's not the case because CSS specificity is dependent on order of inclusion.

Not a big concern right now since this is the only import. I'd be interested in confirming though. Because if it would be duplicated, then this is not a good place to include the theme. We'd be better off creating a wrapper component for ng-select that includes the css.

My first thought still is that the rules would be duplicated.

Reply:
Since this is the only component where ng-select is being used right now I think it's ok. We'll probably use ng-select more in future and then we can move it to a common CSS file or wrap it in a custom component as you said.